README.md

```diff
@@ -15,26 +15,24 @@ popularized by Kubernetes.  Libpod also contains the Pod Manager tool `(Podman)`
 
 ## Overview and scope
 
-At a high level, the scope of libpod and Podman is the following:
+At a high level, the scope of Podman and `libpod` is the following:
 
-* Support multiple image formats including the OCI and Docker image formats.
-* Support for multiple means to download images including trust & image verification.
-* Container image management (managing image layers, overlay filesystems, etc).
-* Full management of container lifecycle.
-* Support for pods to manage groups of containers together.
+* Support for multiple container image formats, including the OCI and Docker.
+* Full management of those images, including pulling from various sources (including trust and verification), creating (built via Containerfile and Dockerfile or committed from a container), and pushing to registries and other storage backends.
```

Comment:
should it be "via a Containerfile" ?  (added an 'a' before Containerfile)

Reply:
I agree. "via a Containerfile OR Dockerfile".